source/getting-started/onboarding.markdown

```diff
@@ -3,56 +3,53 @@ title: "Onboarding Home Assistant"
 description: "Instructions to get Home Assistant configured."
 ---
 
-Alright, you made it here. The tough part is done.
+After your Home Assistant server is up and running, there are 5 simple steps to log in and see your devices.
 
-- As the last step of the [installation procedure](/installation/) specific to your device, you entered the Home Assistant device's address in your browser's address bar.
-    - Typically, this is `http://homeassistant.local:8123/`.
-    - If you are running an older Windows version or have a stricter network configuration, you might need to access Home Assistant at <a href="http://homeassistant:8123" target="_blank">homeassistant:8123</a> or `http://X.X.X.X:8123` (replace X.X.X.X with your device's IP address).
+**Step 1**: type  <http://homeassistant.local:8123/> into your browser.
+![Home Assistant preparation](/images/getting-started/onboarding_preparing_01_.png)
 
-    - **Result**: You now see the welcome screen. Depending on your hardware, preparation may take a while.
+{% details If that didn’t work %}
+If you are running an older Windows version or have a stricter network configuration, you might need to access Home Assistant at homeassistant:8123 or http://X.X.X.X:8123 (replace X.X.X.X with your device’s IP address).
+{% enddetails %}
 
-    ![Home Assistant preparation](/images/getting-started/onboarding_preparing_01_.png)
+**Step 2:** Click on “Create my Smart Home”.
 
-    - Depending on the preinstalled software version, you might see a slightly different version of the welcome screen.
-      - To view the log files, select the pulsing blue dot.
+This will create an administrator account. It will always be able to change everything in Home Assistant. Enter your name, username, and password and click “Create Account”.
```

Comment:
- Avoid the use of "click", it might be a touch screen or other device.
- Seems like formating of **Create account** got lost?

```suggestion
This will create an administrator account. It will always be able to change everything in Home Assistant. Enter your name, username, and password and select **Create Account**.
```